account_statement_operation_rule/README.rst

```diff
@@ -0,0 +1,97 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+Bank Statement Operation Rules
+==============================
+
+This module complements the Reconciliation of the bank statements.  When
+the bank statement matches one or more journal entry for a line and
+there is a remaining balance, Odoo proposes you to click on buttons that
+will generate write-off entries according to pre-configured *Statement
+Operation Templates*. The aim of this module is to automatically click
+for you on these buttons (i.e. create the write-off journal entries)
+when some rules are respected, rules that you can configure.
+
+It contains 2 types of rule (but can be extended with additional rules),
+described below:
+
+Roundings
```

Comment:
This is a definition list, not a bullet points list (http://docutils.sourceforge.net/docs/user/rst/quickref.html#definition-lists).